### DISCUSSION TITLE
Automatically link GitHub release

### DISCUSSION THREAD

**Discussion by sliekens** [adamralph/minver#944] [Show and tell]
Hi,

I wanted my NuGet packages to have a link to the corresponding tag/release page on GitHub. I couldn't find any examples, so I came up with my own solution to automate it.

I used an MSBuild target which runs after MinVer to set the `PackageReleaseNotes` property.

``` xml
<Project Sdk="Microsoft.NET.Sdk">

  <Target Name="LinkPackageReleaseNotes" AfterTargets="MinVer">
    <PropertyGroup>
      <PackageReleaseNotes>https://github.com/sliekens/gw2sdk/releases/tag/$(MinVerTagPrefix)$(Version)</PackageReleaseNotes>
    </PropertyGroup>
  </Target>

</Project>
```

One minor issue is that the Target runs on every commit, not just tagged commits. I don't know how to solve that without forking out to Git CLI.

**Comment by adamralph** [comment]
> One minor issue is that the Target runs on every commit, not just tagged commits.

Why is that an issue? If you are releasing the package, then you will have tagged the commit. If you are not releasing the package then, the broken link will never be seen.

**Reply by sliekens** [reply]
In my setup, every commit is packaged and pushed to my internal package feed, so internally the broken link is visible. But it's really a very minor problem.

**Reply by sliekens** [reply]
The solution which is so obvious that I missed it, is to slap a Condition on the Target that needs to be set from the command line

dotnet pack /p:LinkReleaseNotes=true
